lib/bot/issue_manager.rb

```diff
@@ -144,6 +144,19 @@ def add_labels(command_value, author, issue)
     end
     if !valid_labels.empty?
       issue.add_labels(valid_labels)
+      notify_on_label(issue, valid_labels)
+    end
+  end
+
+  def notify_on_label(issue, labels)
+    labels.each do |label|
+      if Settings.label_notifier[label]
+        message = "cc "
+        Settings.label_notifier[label].each do |contact|
```

Comment:
There's no point in having an empty contact list (just remove the label).
about the second syntax - I know it, but to me it's really hard to read.